buildSrc/build.gradle.kts

```diff
@@ -38,7 +38,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.61")
```

Comment:
You need to match this with Kotlin's version below: `1.4.0`

Reply:
I cannot do this as the `kotlin-dsl` plugin still requires `1.3.61`